src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -245,12 +258,24 @@ class WorkspaceMembersPage extends React.Component {
             return <Navigation.DismissModal />;
         }
         const policyEmployeeList = lodashGet(this.props, 'policy.employeeList', []);
-        const removableMembers = _.without(this.props.policy.employeeList, this.props.session.email, this.props.policy.owner);
-        const data = _.chain(policyEmployeeList)
-            .map(email => this.props.personalDetails[email])
-            .filter()
-            .sortBy(person => person.displayName.toLowerCase())
-            .value();
+        const pendingInvitationList = lodashGet(this.props, 'policy.pendingInvitations', []);
+        const removableMembers = _.without(
```

Comment:
All of the following consts except for `data` can be declaring in one line and without the blank lines between them: 

```diff 
             return <Navigation.DismissModal />;
         }
+
+        const policyID = lodashGet(this.props.route, 'params.policyID');
         const policyEmployeeList = lodashGet(this.props, 'policy.employeeList', []);
         const pendingInvitationList = lodashGet(this.props, 'policy.pendingInvitations', []);
-        const removableMembers = _.without(
-            policyEmployeeList.concat(pendingInvitationList),
-            this.props.session.email,
-            this.props.policy.owner,
-        );
-
-        const employeeListData = _.map(policyEmployeeList,
-            email => this.props.personalDetails[email]);
-
-        const pendingInvitationsData = _.map(pendingInvitationList,
-            email => ({...this.props.personalDetails[email], pending: true}));
-
+        const removableMembers = _.without(policyEmployeeList.concat(pendingInvitationList), this.props.session.email, this.props.policy.owner);
+        const employeeListData = _.map(policyEmployeeList, email => this.props.personalDetails[email]);
+        const pendingInvitationsData = _.map(pendingInvitationList, email => ({...this.props.personalDetails[email], pending: true}));
         const data = _.sortBy(
             employeeListData.concat(pendingInvitationsData),
             person => person && person.displayName.toLowerCase(),
         );
```

Reply:
Okay